packages/cashier/src/components/cashier-locked/cashier-locked.scss

```diff
@@ -0,0 +1,7 @@
+.cashier-locked {
+    margin: 0;
+
+    .cashier-empty-state__action {
```

Comment:
This supposed to be nested ?🤔

Reply:
there is design mismatch between cashier-locked and the empty state generic component. so i need to change the style for empty state component only for the cashier locked , thats why its here. if i put it outside it will change the empty state styles in other places 👀

Reply:
okok 😸